docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Database Access with Redshift Serverless on AWS
+description: How to configure Teleport Database Access with AWS Redshift Serverless.
+---
+
+This guide will help you to:
+
+- Set up Teleport to access your AWS Redshift Serverless instances.
```

Comment:
nitpick: not a big deal but since this is serverless it's a little bit weird to me that we say `instances` here. maybe `databases`, or `workgroups`?

Reply:
Sure, I was at a loss for terminology here.